src/NLog/Common/InternalLogger.cs

```diff
@@ -288,7 +288,7 @@ private static string FormatMessage([CanBeNull]Exception ex, LogLevel level, str
             var formattedMessage =
                 (args == null) ? message : string.Format(CultureInfo.InvariantCulture, message, args);
 
-            var builder = new StringBuilder(formattedMessage.Length + TimeStampFormat.Length + 8);
+            var builder = new StringBuilder(formattedMessage.Length + TimeStampFormat.Length + ex?.ToString()?.Length ?? 0 + 25);
```

Comment:
0 + 25? Should be 8+25?

or 0 + 0+ 0+ 0+ 0+ 0+ 0+ 0 + 25 :P

Reply:
```formattedMessage.Length``` and ```TimeStampFormat.Length``` (constant length) should never be 0. ```ex?.ToString()?.Length ?? 0``` might be zero and the rest of white-spaces the literal 'Exception' and the LogLevel as a string sum up, if I recall correctly, to 23 characters.

To be honest, I am not certain if pre-calculating the length of the expected string in such detail for optimisation is positive. 

What do you think?

Reply:
true, pre-optimized could be done later. 

`0 + 25` looks just odd. Its 25 then.